### PR TITLE
feat: add sixty five percent layout

### PR DESCRIPTION
**Pull request by EugeneTeu**
TSIA 

Capture

### CODE REVIEW

**Comment by yangshun** [comment]
Whoa thanks @EugeneTeu. Can you help add a toggle/selector below the keyboard to change between 80% and 65%? Won't be nice to show two React keyboards in a row especially when the difference is not that big.

**Comment by yangshun** [comment]
> TSIA

I miss this. LOL

**Comment by EugeneTeu** [comment]
> 

Sure, that was the idea i had in mind initially. Will add soon!

**Comment by EugeneTeu** [comment]
https://user-images.githubusercontent.com/43085751/236676492-aa75ca98-f1f2-48db-8389-e8caddca4f9a.mov

meh not my best work, the keyboard shifts in place when it toggles the size selector....

File: src/components/KeyboardRow.tsx [hunk 1]
```diff
@@ -16,3 +16,15 @@ export default function KeyboardRow({ children }: Props) {
     </div>
   );
 }
+
+export function KeyboardRowSixtyFive({ children }: Props) {
```
Comment (yangshun) [inline src/components/KeyboardRow.tsx:20]: Is this meant to be here?

File: src/app/page.tsx [hunk 2]
```diff
@@ -20,6 +20,7 @@ import ReactMark from '~/components/logos/react.svg';
 import SvelteMark from '~/components/logos/svelte.svg';
 import VueMark from '~/components/logos/vue.svg';
 import { ArrowDownIcon, ArrowUpIcon } from 'lucide-react';
+import ReactSixtyFiveKeyboard from '~/components/keyboards/react/ReactSixtyFiveKeyboard';
```
Comment (yangshun) [inline src/app/page.tsx:23]: Doesn't seem needed anymore

**Review by yangshun — COMMENTED** [review]
Cool! Thanks for this! I can fix the issues, just need to confirm the comments whether the additions are still needed.

File: src/app/page.tsx (hunk 2 above)
Comment (EugeneTeu) [reply src/app/page.tsx:23]: oh my yea sorry didnt have time to open my fun coding laptop lol! glad to see it merged

**Pull request merged**